pom.xml

```diff
@@ -311,6 +307,12 @@
             <artifactId>CommandHandler</artifactId>
             <version>11</version>
         </dependency>
+        <!-- End of CommandHandler Dependency -->
+        <dependency>
+            <groupId>net.minidev</groupId>
+            <artifactId>json-smart</artifactId>
+            <version>2.3</version>
```

Comment:
I'm not sure if version 2.3 is compatible. See here, https://netplex.github.io/json-smart/, it says version 2.X is a new interface.

Reply:
I dont facing any issues with `mv version` or paste services. Any known concern about using v2?

Reply:
If it works the same then I guess there's no issue.